wolfcrypt/src/asn.c

```diff
@@ -10562,20 +10562,28 @@ int PemToDer(const unsigned char* buff, long longSz, int type,
             }
             /* decrypt the key */
             else {
-                ret = wc_BufferKeyDecrypt(info, der->buffer, der->length,
-                    (byte*)password, passwordSz, WC_MD5);
+                int length;
+                word32 inOutIdx = 0;
+                if ((passwordSz == 0) &&
+                    (GetSequence(der->buffer, &inOutIdx, &length,
```

Comment:
Please add a short comment here for why this was added. Also curious why not always do a GetSequence check to ensure its always a valid ASN header?

Reply:
The sequence has already been parsed by `wc_EncryptedInfoParse()`, so the call to `GetSequence()` is not actually needed.